content/html/concepts/tags/terms/font/font.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: '<font>'
+Description: 'Used to be used to set the font characteristics of a text. This is now deprecated.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Style'
+  - 'Typography'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<font>`** tag was used to set the font characteristics of a text. It is now deprecated. [CSS](https://www.codecademy.com/resources/docs/css) properties like the [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property should be used instead. 
+
+## Syntax
+
+Any combination of the following three attributes could be used in the `<font>` tag, at least one attribute is required:
```

Comment:
```suggestion
Any combination of the following three attributes could be used in the `<font>` tag, at least any one attribute is required:
```
I feel like it may sound like to some people that 1 is required but it doesn't say it says. Should we change it to this?

Reply:
sure, we could do that.